tests/testsuite/cargo_add/empty_dep_name/stderr.log

```diff
@@ -1 +1 @@
-error: dependency name cannot be empty
+error: package name cannot be empty
```

Comment:
As the dependency name is generally a package name, I figured this might be a worthwhile trade off compared to adding a `DependencyName` type to interact with `PackageName`.

Thoughts?